llvm/test/MC/WebAssembly/reference-types.s

```diff
@@ -27,6 +27,21 @@ ref_null_test:
   drop
   end_function
 
+# CHECK-LABEL: ref_test_test:
+# CHECK: ref.null_func   # encoding: [0xd0,0x70]
+# CHECK: ref.test () -> () # encoding: [0xfb,0x14,0x80'A',0x80'A',0x80'A',0x80'A',A]
```

Comment:
Why call it `ref.test_func` in the assembly if what it generates is `ref.test`?

Is `ref.test_func` a real wasm instruction?  If not then we should probably think twice before adding it, or at least document it somewhere I suppose?   What is the rational for not just using `ref.test`?

Reply:
Makes sense to me. I originally called it `ref.test` but then changed it to `ref.test_func` to be similar to `ref.null_func`, but now looking at it we have `ref.is_null` which works on both `exernref` and `funcref` and is the most similar.

Reply:
Updated it.

Reply:
I didn't know about `ref.null_func`  is that also something llvm specifc?

Reply:
Yes it's also llvm specific. It translates to `ref.null funcref` in the text format.

Reply:
Interesting... I wonder why that was added that way.   I guess this stuff is all very experimental so changing it is probably not a huge deal, but we should try to be consistent I guess.   @dschuff @tlively WDYT?

Reply:
It seems like it was initially added like `ref.null func` or `ref.null extern` but then changed to this in https://reviews.llvm.org/D114979 in order to simplify the text format.